app/assets/stylesheets/app/documents.css.scss

```diff
@@ -0,0 +1,5 @@
+tr.hidden-folder {
+  td, a:not(.btn) {
+    color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.